rdr/container_image/Dockerfile

```diff
@@ -4,5 +4,9 @@ RUN git clone https://github.com/avati/arequal.git
 WORKDIR arequal
 RUN ./autogen.sh ; ./configure ; make ; make install
 WORKDIR /
+ADD https://raw.githubusercontent.com/red-hat-storage/ocs-workloads/master/rdr/busybox/scripts/simple_io_integrity_check.sh /mnt/simple_io_integrity_check.sh
+RUN chmod +x /mnt/simple_io_integrity_check.sh
```

Comment:
if we do set properly the permission on those files in repository - I think the permission will be carried on via `COPY` as well inside the image - so if my assumption is correct - you can remove those `chmod +x`  RUN Commands

Reply:
will check and test it for now will keep as it is